tests/suite/zar/s3/import.yaml

```diff
@@ -0,0 +1,27 @@
+script: |
+  export AWS_REGION=us-east-2 AWS_ACCESS_KEY_ID=minioadmin \
+        AWS_SECRET_ACCESS_KEY=minioadmin AWS_S3_ENDPOINT=http://localhost:9000
+  mkdir -p data/bucket
+  minio server --quiet --address "localhost:9000" ./data &> /dev/null &
+  pid=$!
+  until nc -vzw 2 localhost 9000; do sleep 2; done
+  zar import -s 20KiB -R ./root -data s3://bucket/zartest babble.tzng
+  echo ===
+  zar ls -R ./root
+  echo ===
+  zar ls -relative -R ./root
+  kill -KILL $pid &> /dev/null
```

Comment:
`trap "kill -9 $!" EXIT` immediately after the background command instead?

Reply:
This would be nice I'll try this out.